f2/src/components/warning-banner/index.js

```diff
@@ -2,6 +2,7 @@
 /** @jsx jsx **/
 import { jsx } from '@emotion/core'
 import PropTypes from 'prop-types'
+import { Trans } from '@lingui/macro'
 import { Alert, AlertIcon, AlertDescription } from '@chakra-ui/core'
 
 export const WarningBanner = props => {
```

Comment:
remove message from the props list

Reply:
https://github.com/cds-snc/report-a-cybercrime/blob/713a5b80399400a41435b4209d45f1d6ee8aabd1/f2/src/components/warning-banner/index.js#L8

Reply:
oh no, it's fixed now :)